ChangeLog.md

```diff
@@ -3,6 +3,308 @@
 Notable changes to the FreeOrion project will be documented in this file.
 
 
+## [v0.4.6] - 2016-09-10
+
+
+### Key Changes
+
+- Reworked supply mechanics
+- Various GUI improvements
+ - GUI responsiveness / FPS improvements
+ - Made GUI more configurable
+ - New/better graphics
+ - Lot of other small improvements to handling/display of things
```

Comment:
Basically agree, but as the FPS improvements are already mentioned above, I'd drop that specific bit (and just leave "...and system resource usage").

Another thing I tried to cover by using the term "handling" are the many small additions to GUI functionality (like dismissing sensor ghosts etc.). So maybe phrase it like this:
```
Many small additions/changes to GUI rendering, layout, functionality, customization options, and displayed information that collectively substantially improve usability, responsiveness, and system resource usage.
```
...?